fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -4554,7 +4563,7 @@ private static String genLongUrl(final String prefix, final int numChars) {
 
     @Test
     public void testLongIdentifier1() throws IOException {
-        final String url = genLongUrl(serverAddress,476);
+        final String url = genLongUrl(serverAddress,475);
```

Comment:
What's this change about?  Also I'm surprised this works since presumably we're posting to a non-existent resource no?  Just curious.

Reply:
I don't know understand what was going on. For some reason that test started failing. I thought maybe the random id was a character longer for some reason....